src/repositories/media.repository.ts

```diff
@@ -12,6 +12,25 @@ import Chance = require('chance');
 import { MediaRepositoryCommentResponseRootObject } from '../responses/media.repository.configure.response';
 
 export class MediaRepository extends Repository {
+  public async info({
+    mediaId,
```

Comment:
Why did you decide to pass object here? Maybe it will be better to use just string?

Reply:
Thanks will change!